packages/ui-components/src/browser-icon.jsx

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash'
+import React from 'React'
+
+const edgeRe = /^edge/
+const firefoxRe = /^firefox/
+
+const logoPaths = {
+  canary: require('browser-logos/src/chrome-canary/chrome-canary_32x32.png'),
+  chrome: require('browser-logos/src/chrome/chrome_32x32.png'),
+  chromium: require('browser-logos/src/chromium/chromium_32x32.png'),
+  edge: require('browser-logos/src/edge/edge_32x32.png'),
+  electron: require('browser-logos/src/electron/electron_32x32.png'),
+  firefox: require('browser-logos/src/firefox/firefox_32x32.png'),
+}
+
+const logoPath = (browserName) => {
+  const browserKey = () => {
+    if (edgeRe.test(browserName)) {
+      return 'edge'
+    }
+
+    if (firefoxRe.test(browserName)) {
+      return 'firefox'
+    }
+
```

Comment:
Edge Beta, Firefox Dev Edition, etc. have their own icons we can display, just like Chrome Canary. Any reason not to?